src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,38 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.10.0
+5 July 2023
+
+## Removal Notice
+
+Kubernetes 1.23 is no longer supported.
+
+### Features
+
+- Now you can select an expiration period when creating a new Personal Access Token. <!-- 6690 -->
+- Administrators have the ability to wake up, put to sleep, or mark any preview environment as keep-awake. <!-- 6662, 6622 -->
+- Upgraded ingress-nginx to [4.6](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.7.0). <!-- 6625 -->
+
+### Improvements
+
+- Kubernetes Annotations can now be added to: <!-- 6689, 6668 -->
+  - Default self-signed wildcard certificate via `.Values.wildcardCertificate.annotations`.
+  - Internal self-signed webhook certificate via `.Values.webhook.internalCertificate.annotations`.
+  - Dockerconfig via `.Values.secret.annotations`.
+- Kubernetes Labels can now be added to: <!-- 6668 -->
+  - Dockerconfig via `.Values.secret.labels`.
+
+### Bugfixes
+
+- Fixed an installation step warning that was displayed incorrectly when using the default `.Values.subdomain` (`localtest.me`). <!-- 6717 -->
+- Cert-manager annotations are now allowed in developer ingresses. <!-- 6697 -->
+- When a license is not specified, the Starter plan is displayed instead of Unknown. <!-- 6671 -->
+- Upgraded the Okteto CLI version to [2.16.5](https://github.com/okteto/okteto/releases/tag/2.16.5). <!-- 6762, 6764 -->
+  - When using `okteto deploy --remote ...` with private certificates, internal registry requests didn't use them correctly. <!-- okteto/3743, okteto/3758 -->
```

Comment:
```suggestion
  - When using `okteto deploy --remote ...` with private certificates, internal registry requests will now work correctly. <!-- okteto/3743, okteto/3758 -->
```

Non-blocking suggestion to focus more on the positive outcome vs the negative impact from before